packages/sync/src/class-replicastore.php

```diff
@@ -870,6 +870,10 @@ public function delete_site_option( $option ) {
 	 * @return array Array of terms.
 	 */
 	public function get_terms( $taxonomy ) {
+		$t = $this->ensure_taxonomy( $taxonomy );
+		if ( ! $t || is_wp_error( $t ) ) {
+			return $t;
```

Comment:
Assuming we're not worried about breaking any callers, the `@return` on line 870 needs updating to reflect this change.